src/frontend/komponenter/ManuellJournalfør/KnyttJournalpostTilBehandling.tsx

```diff
@@ -103,23 +102,19 @@ export const KnyttJournalpostTilBehandling: React.FC = () => {
                                             datoformat.DATO_FORKORTTET
                                         )}
                                     </td>
-                                    <td>{formaterTilKunFørstBokstavStor(behandling.årsak)}</td>
+                                    <td>{behandlingÅrsak[behandling.årsak]}</td>
                                     <BehandlingstypeTd>
-                                        {formaterTilKunFørstBokstavStor(behandling.type)}
+                                        {behandlingstyper[behandling.type].navn}
```

Comment:
Noen grunn til at behandlingÅrsak er en Record mens behandlingstyper er INøkkelPar?

Reply:
Jeg har ikke sett noe på det i denne PRen, men vi bruke INøkkelPar før vi skrev om til record. Bør nok ta en sjau å fjerne INøkkelPar etterhvert :)